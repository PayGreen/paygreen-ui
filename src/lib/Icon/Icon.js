import React from 'react';
import IconWrapper from './IconWrapper';
import {
  ArrowBottom,
  ArrowCircleLeft,
  ArrowCircleLeftBold,
  ArrowCircleRightBold,
  ArrowCircleRight,
  ArrowLeft,
  ArrowLongBottom,
  ArrowLongLeft,
  ArrowLongRight,
  ArrowLongTop,
  ArrowRight,
  ArrowRoundBottomLeft,
  ArrowRoundBottomRight,
  ArrowRoundTopLeft,
  ArrowRoundTopRight,
  ArrowTop,
  BalloonBold,
  Balloon,
  Bank,
  BeakerBold,
  Beaker,
  BellBold,
  Bell,
  BulbBold,
  Bulb,
  CardsBold,
  Cards,
  Car,
  Cart,
  CautionBold,
  Caution,
  Change,
  CheckBold,
  Checkbox,
  Check,
  ClockBold,
  Clock,
  Code,
  Coins,
  Contract,
  CrossBold,
  Cross,
  Download,
  Empty,
  Eye,
  Factory,
  Globe,
  Heart,
  Hourglass,
  Information,
  Leaf,
  LetterBold,
  Letter,
  LinkBold,
  Link,
  List,
  Lock,
  Marker,
  Meal,
  Megaphone,
  MenuBold,
  MenuDots,
  Menu,
  Meter,
  Minus,
  Moon,
  Moove,
  Organization,
  Out,
  Paperclip,
  Pen,
  Phone,
  Picture,
  Plane,
  Plus,
  Pointer,
  Popin,
  QuestionBold,
  Question,
  QuoteLeft,
  QuoteRight,
  RefreshBold,
  Refresh,
  Satchel,
  Schedule,
  SearchBold,
  Search,
  Shop,
  Sun,
  Support,
  Trees,
  Trophy,
  Truck,
  Tv,
  UserBold,
  Users,
  User,
  Wallet,
  Weight,
  Wheels,
} from './components';

export const ArrowBottomIcon = (props) => <IconWrapper {...props}><ArrowBottom /></IconWrapper>
export const ArrowCircleLeftIcon = (props) => <IconWrapper {...props}><ArrowCircleLeft /></IconWrapper>
export const ArrowCircleLeftBoldIcon = (props) => <IconWrapper {...props}><ArrowCircleLeftBold /></IconWrapper>
export const ArrowCircleRightBoldIcon = (props) => <IconWrapper {...props}><ArrowCircleRightBold /></IconWrapper>
export const ArrowCircleRightIcon = (props) => <IconWrapper {...props}><ArrowCircleRight /></IconWrapper>
export const ArrowLeftIcon = (props) => <IconWrapper {...props}><ArrowLeft /></IconWrapper>
export const ArrowLongBottomIcon = (props) => <IconWrapper {...props}><ArrowLongBottom /></IconWrapper>
export const ArrowLongLeftIcon = (props) => <IconWrapper {...props}><ArrowLongLeft /></IconWrapper>
export const ArrowLongRightIcon = (props) => <IconWrapper {...props}><ArrowLongRight /></IconWrapper>
export const ArrowLongTopIcon = (props) => <IconWrapper {...props}><ArrowLongTop /></IconWrapper>
export const ArrowRightIcon = (props) => <IconWrapper {...props}><ArrowRight /></IconWrapper>
export const ArrowRoundBottomLeftIcon = (props) => <IconWrapper {...props}><ArrowRoundBottomLeft /></IconWrapper>
export const ArrowRoundBottomRightIcon = (props) => <IconWrapper {...props}><ArrowRoundBottomRight /></IconWrapper>
export const ArrowRoundTopLeftIcon = (props) => <IconWrapper {...props}><ArrowRoundTopLeft /></IconWrapper>
export const ArrowRoundTopRightIcon = (props) => <IconWrapper {...props}><ArrowRoundTopRight /></IconWrapper>
export const ArrowTopIcon = (props) => <IconWrapper {...props}><ArrowTop /></IconWrapper>
export const BalloonBoldIcon = (props) => <IconWrapper {...props}><BalloonBold /></IconWrapper>
export const BalloonIcon = (props) => <IconWrapper {...props}><Balloon /></IconWrapper>
export const BankIcon = (props) => <IconWrapper {...props}><Bank /></IconWrapper>
export const BeakerBoldIcon = (props) => <IconWrapper {...props}><BeakerBold /></IconWrapper>
export const BeakerIcon = (props) => <IconWrapper {...props}><Beaker /></IconWrapper>
export const BellBoldIcon = (props) => <IconWrapper {...props}><BellBold /></IconWrapper>
export const BellIcon = (props) => <IconWrapper {...props}><Bell /></IconWrapper>
export const BulbBoldIcon = (props) => <IconWrapper {...props}><BulbBold /></IconWrapper>
export const BulbIcon = (props) => <IconWrapper {...props}><Bulb /></IconWrapper>
export const CardsBoldIcon = (props) => <IconWrapper {...props}><CardsBold /></IconWrapper>
export const CardsIcon = (props) => <IconWrapper {...props}><Cards /></IconWrapper>
export const CarIcon = (props) => <IconWrapper {...props}><Car /></IconWrapper>
export const CartIcon = (props) => <IconWrapper {...props}><Cart /></IconWrapper>
export const CautionBoldIcon = (props) => <IconWrapper {...props}><CautionBold /></IconWrapper>
export const CautionIcon = (props) => <IconWrapper {...props}><Caution /></IconWrapper>
export const ChangeIcon = (props) => <IconWrapper {...props}><Change /></IconWrapper>
export const CheckBoldIcon = (props) => <IconWrapper {...props}><CheckBold /></IconWrapper>
export const CheckboxIcon = (props) => <IconWrapper {...props}><Checkbox /></IconWrapper>
export const CheckIcon = (props) => <IconWrapper {...props}><Check /></IconWrapper>
export const ClockBoldIcon = (props) => <IconWrapper {...props}><ClockBold /></IconWrapper>
export const ClockIcon = (props) => <IconWrapper {...props}><Clock /></IconWrapper>
export const CodeIcon = (props) => <IconWrapper {...props}><Code /></IconWrapper>
export const CoinsIcon = (props) => <IconWrapper {...props}><Coins /></IconWrapper>
export const ContractIcon = (props) => <IconWrapper {...props}><Contract /></IconWrapper>
export const CrossBoldIcon = (props) => <IconWrapper {...props}><CrossBold /></IconWrapper>
export const CrossIcon = (props) => <IconWrapper {...props}><Cross /></IconWrapper>
export const DownloadIcon = (props) => <IconWrapper {...props}><Download /></IconWrapper>
export const EmptyIcon = (props) => <IconWrapper {...props}><Empty /></IconWrapper>
export const EyeIcon = (props) => <IconWrapper {...props}><Eye /></IconWrapper>
export const FactoryIcon = (props) => <IconWrapper {...props}><Factory /></IconWrapper>
export const GlobeIcon = (props) => <IconWrapper {...props}><Globe /></IconWrapper>
export const HeartIcon = (props) => <IconWrapper {...props}><Heart /></IconWrapper>
export const HourglassIcon = (props) => <IconWrapper {...props}><Hourglass /></IconWrapper>
export const InformationIcon = (props) => <IconWrapper {...props}><Information /></IconWrapper>
export const LeafIcon = (props) => <IconWrapper {...props}><Leaf /></IconWrapper>
export const LetterBoldIcon = (props) => <IconWrapper {...props}><LetterBold /></IconWrapper>
export const LetterIcon = (props) => <IconWrapper {...props}><Letter /></IconWrapper>
export const LinkBoldIcon = (props) => <IconWrapper {...props}><LinkBold /></IconWrapper>
export const LinkIcon = (props) => <IconWrapper {...props}><Link /></IconWrapper>
export const ListIcon = (props) => <IconWrapper {...props}><List /></IconWrapper>
export const LockIcon = (props) => <IconWrapper {...props}><Lock /></IconWrapper>
export const MarkerIcon = (props) => <IconWrapper {...props}><Marker /></IconWrapper>
export const MealIcon = (props) => <IconWrapper {...props}><Meal /></IconWrapper>
export const MegaphoneIcon = (props) => <IconWrapper {...props}><Megaphone /></IconWrapper>
export const MenuBoldIcon = (props) => <IconWrapper {...props}><MenuBold /></IconWrapper>
export const MenuDotsIcon = (props) => <IconWrapper {...props}><MenuDots /></IconWrapper>
export const MenuIcon = (props) => <IconWrapper {...props}><Menu /></IconWrapper>
export const MeterIcon = (props) => <IconWrapper {...props}><Meter /></IconWrapper>
export const MinusIcon = (props) => <IconWrapper {...props}><Minus /></IconWrapper>
export const MoonIcon = (props) => <IconWrapper {...props}><Moon /></IconWrapper>
export const MooveIcon = (props) => <IconWrapper {...props}><Moove /></IconWrapper>
export const OrganizationIcon = (props) => <IconWrapper {...props}><Organization /></IconWrapper>
export const OutIcon = (props) => <IconWrapper {...props}><Out /></IconWrapper>
export const PaperclipIcon = (props) => <IconWrapper {...props}><Paperclip /></IconWrapper>
export const PenIcon = (props) => <IconWrapper {...props}><Pen /></IconWrapper>
export const PhoneIcon = (props) => <IconWrapper {...props}><Phone /></IconWrapper>
export const PictureIcon = (props) => <IconWrapper {...props}><Picture /></IconWrapper>
export const PlaneIcon = (props) => <IconWrapper {...props}><Plane /></IconWrapper>
export const PlusIcon = (props) => <IconWrapper {...props}><Plus /></IconWrapper>
export const PointerIcon = (props) => <IconWrapper {...props}><Pointer /></IconWrapper>
export const PopinIcon = (props) => <IconWrapper {...props}><Popin /></IconWrapper>
export const QuestionBoldIcon = (props) => <IconWrapper {...props}><QuestionBold /></IconWrapper>
export const QuestionIcon = (props) => <IconWrapper {...props}><Question /></IconWrapper>
export const QuoteLeftIcon = (props) => <IconWrapper {...props}><QuoteLeft /></IconWrapper>
export const QuoteRightIcon = (props) => <IconWrapper {...props}><QuoteRight /></IconWrapper>
export const RefreshBoldIcon = (props) => <IconWrapper {...props}><RefreshBold /></IconWrapper>
export const RefreshIcon = (props) => <IconWrapper {...props}><Refresh /></IconWrapper>
export const SatchelIcon = (props) => <IconWrapper {...props}><Satchel /></IconWrapper>
export const ScheduleIcon = (props) => <IconWrapper {...props}><Schedule /></IconWrapper>
export const SearchBoldIcon = (props) => <IconWrapper {...props}><SearchBold /></IconWrapper>
export const SearchIcon = (props) => <IconWrapper {...props}><Search /></IconWrapper>
export const ShopIcon = (props) => <IconWrapper {...props}><Shop /></IconWrapper>
export const SunIcon = (props) => <IconWrapper {...props}><Sun /></IconWrapper>
export const SupportIcon = (props) => <IconWrapper {...props}><Support /></IconWrapper>
export const TreesIcon = (props) => <IconWrapper {...props}><Trees /></IconWrapper>
export const TrophyIcon = (props) => <IconWrapper {...props}><Trophy /></IconWrapper>
export const TruckIcon = (props) => <IconWrapper {...props}><Truck /></IconWrapper>
export const TvIcon = (props) => <IconWrapper {...props}><Tv /></IconWrapper>
export const UserBoldIcon = (props) => <IconWrapper {...props}><UserBold /></IconWrapper>
export const UsersIcon = (props) => <IconWrapper {...props}><Users /></IconWrapper>
export const UserIcon = (props) => <IconWrapper {...props}><User /></IconWrapper>
export const WalletIcon = (props) => <IconWrapper {...props}><Wallet /></IconWrapper>
export const WeightIcon = (props) => <IconWrapper {...props}><Weight /></IconWrapper>
export const WheelsIcon = (props) => <IconWrapper {...props}><Wheels /></IconWrapper>
