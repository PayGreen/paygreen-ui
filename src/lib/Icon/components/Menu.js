import React from "react";

const SvgMenu = props => (
  <svg viewBox="0 0 143 143" {...props}>
    <path d="M114.8 45.6H26.2c-1.9 0-3.5-1.6-3.5-3.5s1.6-3.5 3.5-3.5h88.7c1.9 0 3.5 1.6 3.5 3.5s-1.6 3.5-3.6 3.5zM114.8 76.9H26.2c-1.9 0-3.5-1.6-3.5-3.5s1.6-3.5 3.5-3.5h88.7c1.9 0 3.5 1.6 3.5 3.5s-1.6 3.5-3.6 3.5zM114.8 107.9H26.2c-1.9 0-3.5-1.6-3.5-3.5s1.6-3.5 3.5-3.5h88.7c1.9 0 3.5 1.6 3.5 3.5s-1.6 3.5-3.6 3.5z" />
  </svg>
);

export default SvgMenu;
