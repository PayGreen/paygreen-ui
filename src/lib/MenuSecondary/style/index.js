import styled from 'styled-components';
import { transparentize } from 'polished';
import { menuBlock } from '../../MenuPrimary/style/base';
import { closedStyle } from './base';
import { MenuBase } from '../../Menu/style';
import { MenuCloseBase } from '../../MenuClose/style';

const MenuSecondaryBase = styled.div`
    ${menuBlock};

    ul {
        margin: 0;
        padding: ${props => props.theme.space.sm} 0;

        @media (${props => props.theme.query.min.lg}) {
            display: flex;
            padding: 0;
        }

        li {
            list-style-type: none;
            margin: 0;
            padding: 0;

            @media (${props => props.theme.query.min.lg}) {
                display: flex;
                align-items: center;
            }
        }
    }

    @media (${props => props.theme.query.max.lg}) {
        max-height: calc(100vh - ${props => props.theme.grid.header});
        box-shadow: 0 3px 20px ${props => transparentize(
            props.theme.shadow.opacity.sm,
            props.theme.wab.black00
        )};
        
        ${props => props.isOpen ? null : closedStyle};

        ${MenuCloseBase} {
            margin: ${props => props.theme.space.md} auto 0 auto;
        }
    }

    @media (${props => props.theme.query.min.lg}) {
        ${MenuBase} {
            position: relative;
            display: flex;
            align-items: center;
            padding: 0;
        }

        ${MenuCloseBase} {
            display: none;
        }
    }
`;

export { MenuSecondaryBase };