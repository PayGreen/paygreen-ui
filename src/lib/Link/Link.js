import React from 'react';
import PropTypes from 'prop-types';
import {
    colorTypeOptions,
    colorTypeDefault,
    colorPalletOptions,
    colorPalletDefault,
    colorThemeOptions,
    colorThemeDefault,
    greyOptions,
    greyDefault,
    formStatusOptions,
    formStatusDefault,
} from '../../shared/constants';
import { LinkBase } from './style';

const Link = props => {
    return <LinkBase {...props}>{props.children}</LinkBase>;
};

Link.propTypes = {
    colorType: PropTypes.oneOf(Object.values(colorTypeOptions)),
    colorPallet: PropTypes.oneOf(Object.values(colorPalletOptions)),
    colorTheme: PropTypes.oneOf(Object.values(colorThemeOptions)),
    colorWab: PropTypes.oneOf(Object.values(greyOptions)),
    colorStatus: PropTypes.oneOf(Object.values(formStatusOptions)),
    hasUnderline: PropTypes.bool,
    hasHover: PropTypes.bool,
    hasUppercase: PropTypes.bool,
    transition: PropTypes.string,
};

Link.defaultProps = {
    colorType: colorTypeDefault,
    colorPallet: colorPalletDefault,
    colorTheme: colorThemeDefault,
    colorWab: greyDefault,
    colorStatus: formStatusDefault,
    hasUnderline: true,
    hasHover: true,
    hasUppercase: false,
    transition: null,
};

export default Link;
