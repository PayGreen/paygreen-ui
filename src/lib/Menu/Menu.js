import React from 'react';
import PropTypes from 'prop-types';
import { itemHtmlTagOptions, itemHtmlTagDefault } from '../../shared/constants';
import { MenuBase } from './style';

const Menu = props => {
    return (
        <MenuBase as={props.htmlTag} {...props}>
            {props.children}
        </MenuBase>
    );
};

Menu.propTypes = {
    htmlTag: PropTypes.oneOf(Object.values(itemHtmlTagOptions)),
};

Menu.defaultProps = {
    htmlTag: itemHtmlTagDefault,
};

export default Menu;
