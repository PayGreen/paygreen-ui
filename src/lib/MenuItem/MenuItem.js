import React from 'react';
import PropTypes from 'prop-types';
import {
    colorThemeOptions,
    colorThemeDefault,
    alignOptions,
    alignDefault,
} from '../../shared/constants';
import { MenuItemBase } from './style';

const MenuItem = props => {
    return <MenuItemBase {...props}>{props.children}</MenuItemBase>;
};

MenuItem.propTypes = {
    colorTheme: PropTypes.oneOf(Object.values(colorThemeOptions)),
    isMain: PropTypes.bool,
    hasHoverBase: PropTypes.bool,
    hasHoverAlternative: PropTypes.bool,
    hasHoverEmphasis: PropTypes.bool,
    align: PropTypes.oneOf(Object.values(alignOptions)),
};

MenuItem.defaultProps = {
    colorTheme: colorThemeDefault,
    isMain: false,
    hasHoverBase: true,
    hasHoverAlternative: true,
    hasHoverEmphasis: false,
    align: alignDefault,
};

export default MenuItem;
