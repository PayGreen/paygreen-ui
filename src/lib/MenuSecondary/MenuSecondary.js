import React from 'react';
import PropTypes from 'prop-types';
import {
    dotStyleOptions,
    iconSizeOptions,
    spaceOptions,
    colorPalletOptions,
    greyOptions,
} from '../../shared/constants';
import Dot from '../Dot/Dot';
import { MenuSecondaryBase } from './style';

const MenuSecondary = props => {
    return (
        <MenuSecondaryBase {...props}>
            <ul className="menu">
                {React.Children.map(props.children, (child, index) =>
                    child ? (
                        <li key={index} className="menuElement">
                            {index ? (
                                <Dot
                                    theme={props.theme} // not necessary, only needed for tests
                                    className="hideOnSmallScreen"
                                    dotStyle={dotStyleOptions.slashes}
                                    dotSize={iconSizeOptions.lg}
                                    colorPallet={colorPalletOptions.wab}
                                    colorWab={greyOptions.grey10}
                                    marginLeft={spaceOptions.xs}
                                    marginRight={spaceOptions.sm}
                                />
                            ) : null}
                            
                            {child}
                        </li>
                    ) : null,
                )}
            </ul>

            {props.closeComponent ? props.closeComponent : null}
        </MenuSecondaryBase>
    );
};

MenuSecondary.propTypes = {
    isOpen: PropTypes.bool,
    closeComponent: PropTypes.element,
};

MenuSecondary.defaultProps = {
    isOpen: PropTypes.false,
};

export default MenuSecondary;
