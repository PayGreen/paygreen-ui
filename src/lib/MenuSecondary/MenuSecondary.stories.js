import React from 'react';
import { storiesOf } from '@storybook/react';
import { withKnobs, boolean } from '@storybook/addon-knobs';
import {
    folder,
    iconSizeOptions,
    itemHtmlTagOptions,
} from '../../shared/constants';
import labels from '../../shared/labels';
import { ShopIcon, OrganizationIcon } from '../Icon/Icon';
import Link from '../Link/Link';
import MenuItem from '../MenuItem/MenuItem';
import MenuList from '../MenuList/MenuList';
import MenuListItem from '../MenuListItem/MenuListItem';
import Menu from '../Menu/Menu';
import MenuClose from '../MenuClose/MenuClose';
import MenuSecondary from './MenuSecondary';

storiesOf(folder.nav + 'MenuSecondary', module)
    .addDecorator(withKnobs)
    .add('MenuSecondary', () => (
        <MenuSecondary
            isOpen={boolean(labels.isOpen, true)}
            closeComponent={<MenuClose />}
        >
            <Menu htmlTag={itemHtmlTagOptions.div}>
                <a href="#">
                    <MenuItem>Login</MenuItem>
                </a>
            </Menu>

            <Menu htmlTag={itemHtmlTagOptions.div}>
                <MenuItem isClickable={false}>Subscribe</MenuItem>

                <MenuList>
                    <a href="#">
                        <MenuListItem>
                            <ShopIcon iconSize={iconSizeOptions.lg} />

                            <Link>As shop</Link>
                        </MenuListItem>
                    </a>

                    <a href="#">
                        <MenuListItem>
                            <OrganizationIcon iconSize={iconSizeOptions.lg} />

                            <Link>As organization</Link>
                        </MenuListItem>
                    </a>
                </MenuList>
            </Menu>
        </MenuSecondary>
    ));
