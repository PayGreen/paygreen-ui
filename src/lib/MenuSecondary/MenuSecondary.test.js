import React from 'react';
import TestRenderer from 'react-test-renderer';
import { ThemeDefault } from '../../theme';
import { LockIcon } from '../Icon/Icon';
import Link from '../Link/Link';
import MenuItem from '../MenuItem/MenuItem';
import Menu from '../Menu/Menu';
import MenuSecondary from './MenuSecondary';

it('renders without crashing', () => {
    const element = TestRenderer.create(
        <MenuSecondary theme={ThemeDefault} isOpen={true}>
            <Menu theme={ThemeDefault}>
                <a href="#">
                    <MenuItem
                        theme={ThemeDefault}
                        isMain={true}
                    >
                        <LockIcon
                            theme={ThemeDefault}
                        />

                        <Link
                            theme={ThemeDefault}
                            underline={false}
                            uppercase={true}
                        >
                            Login
                        </Link>
                    </MenuItem>
                </a>
            </Menu>
        </MenuSecondary>
    );
    expect(element.toJSON()).toMatchSnapshot();
});