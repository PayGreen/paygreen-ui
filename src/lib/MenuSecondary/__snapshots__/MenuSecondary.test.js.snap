// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="style__MenuSecondaryBase-lzoq71-0 GxLgA"
>
  <ul
    className="menu"
  >
    <li
      className="menuElement"
    >
      <div
        className="style__MenuBase-sc-6r1iet-0 EOKtT"
      >
        <a
          href="#"
        >
          <div
            className="style__MenuItemBase-sc-4wfueo-0 kQDKBR"
          >
            <span
              className="style__IconBase-ak3679-0 weCXC icon"
            >
              <svg
                viewBox="0 0 143 143"
              >
                <path
                  d="M95.8 59.3V45.9c0-14.9-10.5-25.8-24.9-25.8-14.3 0-24.7 10.9-24.7 25.8v13.4c-8.5.2-15.3 7.1-15.3 15.6v31.8c0 8.6 7 15.6 15.6 15.6h48.9c8.6 0 15.6-7 15.6-15.5V74.9c0-8.5-6.8-15.4-15.2-15.6zM70.9 27.1c10.5 0 17.9 7.7 17.9 18.8v13.4H53.2V45.9c0-11.1 7.3-18.8 17.7-18.8zm33.1 79.7c0 4.7-3.9 8.5-8.6 8.5H46.5c-4.7 0-8.6-3.9-8.6-8.6V74.9c0-4.7 3.9-8.6 8.6-8.6h48.9c4.7 0 8.6 3.9 8.6 8.6v31.9z"
                />
              </svg>
            </span>
            Login
          </div>
        </a>
      </div>
    </li>
  </ul>
</div>
`;
